en/micro-integrator/docs/setup/brokers/configure-with-rabbitMQ.md

```diff
@@ -18,8 +18,6 @@ parameter.username = "guest"
 parameter.password = "guest"
 ```
 
-Download the [amqp-client-5.7.0.jar](https://www.rabbitmq.com/java-client.html) and copy it into `MI_HOME/lib` directory.
-
 ## Enabling SSL
```

Comment:
SSL configs need to change with https://github.com/wso2/micro-integrator/pull/1053